nautobot_golden_config/jobs.py

```diff
@@ -98,9 +98,14 @@ def run(self, data, commit):  # pylint: disable=too-many-branches
 
         _ = [
             git_wrapper(self, repo, "intended")
-            for repo in GoldenConfigSetting.objects.first().intended_repository.all()
+            for repo in [
+                gcs.intended_repository for gcs in GoldenConfigSetting.objects.all() if gcs.intended_repository
+            ]
+        ]
+        _ = [
+            git_wrapper(self, repo, "backup")
```

Comment:
Does it make sense to push all this logic into the function `git_wrapper`?

Reply:
Not sure if I understand correctly, You mean to:
- simplify `git_wrapper()` method
- remove iteration (`for repo in [ ... `) ?
?

Reply:
Fixed